.github/workflows/azure-dev.yml

```diff
@@ -57,12 +57,6 @@ jobs:
                   cd ..
                   npm run test
                   cd ..
-            - name: Find and Replace DB_CONNECTION_TOKEN
-              uses: cschleiden/replace-tokens@v1
```

Comment:
I'm not sure about using another open-source repo (with only 3 stars). Maybe ask Mark if this is okay or not?
I looked at the src files & it's not alot of code, so you could replicate it in your own repo.

Reply:
I talked to Mark and Stephane. Stephane is registering our use of this open source repo. He said it's okay for us to use it in our pipeline.

Reply:
Okay, sounds good!